File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        vector<int> x;
        if(nums.size()==0 || nums.size()==1)
            return nums.size();
        for(int i=0;i<nums.size()-1;i++){
            if(nums[i]!=nums[i+1])
                x.push_back(nums[i]);
        }
        x.push_back(nums[nums.size()-1]);
        nums=x;
        return x.size();
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
       var set=new HashSet<int>(nums);
        int i=0;
        foreach(var num in set){
            nums[i]=num;
            i++;
        }
        Array.Resize(ref nums,set.Count);
        return nums.Length;
    }
}